resources/lang/en/beatmaps.php

```diff
@@ -64,6 +63,12 @@
             'in_timeline' => 'To mod multiple timestamps, post multiple times (one post per timestamp).',
         ],
 
+        'message_placeholder' => [
+            'general' => 'Type here to post to General (:version)',
+            'generalAll' => 'Type here to post to General (All difficulties) ',
+            'timeline' => 'Type here to Timeline (:version)'
```

Comment:
how to timeline ( ･_･)a

(looks like missing some words and a trailing comma)

Reply:
I got distracted 🔪